test/e2e/tests/navigate-transactions.spec.js

```diff
@@ -0,0 +1,251 @@
+const { strict: assert } = require('assert');
+const { withFixtures } = require('../helpers');
+
+describe('Navigate transactions', function () {
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: 25000000000000000000,
+      },
+    ],
+  };
+  it('should navigate the unapproved transactions', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'navigate-transactions',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
```

Comment:
Nit: I've noticed in our E2E's that we reuse a values like this password, the secret key, etc.  Should we, maybe in a different PR, create a `constants` file and use vars from there?

Reply:
that's a fantastic idea.

Reply:
💯 - i'll revisit that as a separate piece of work